Rakefile

```diff
@@ -29,6 +29,7 @@ task :post_unreleased_changes do
   require 'excon'
 
   logging_exceptions('post-unreleased-changes.log') do |logger|
+    next unless Fastlane::Bot.should_notify_slack_message_not_that_important_though?
```

Comment:
`should_send_trivial_slack_notification`?

Reply:
I'm not good with naming, that is a good one 👍